### ISSUE TITLE
Column mismatch on export

### ISSUE THREAD

**Issue by aklaver** [kennethreitz/records#39]
records==0.2.0
Postgres 9.4.6

Table schema:

```
postgres@production=# \d cell_per
                          Table "public.cell_per"
     Column     |              Type              |        Modifiers         
----------------+--------------------------------+--------------------------
 line_id        | integer                        | not null
 category       | character varying(10)          | not null
 cell_per       | integer                        | not null
 ts_insert      | timestamp(0) without time zone | default now()
 ts_update      | timestamp(0) without time zone | 
 user_insert    | character varying(20)          | default "session_user"()
 user_update    | character varying(20)          | 
 plant_type     | character varying(6)           | not null
 season         | character varying(9)           | not null
 short_category | text                           | not null
Indexes:
    "cell_per_pkey" PRIMARY KEY, btree (line_id)
Foreign-key constraints:
    "fk_category" FOREIGN KEY (category) REFERENCES category(category) ON UPDATE CASCADE
Triggers:
    cell_per_ts_update BEFORE UPDATE ON cell_per FOR EACH ROW EXECUTE PROCEDURE ts_update()
    cell_per_user_update BEFORE UPDATE ON cell_per FOR EACH ROW EXECUTE PROCEDURE user_update()
```
Query:
```
db = records.Database('postgresql://aklaver:@localhost/production')
rs = db.query('select * from cell_per')

>>> rs[0].line_id 
4
```
```
>>> print rs.export('csv')
line_id,category,cell_per,ts_insert,ts_update,user_insert,user_update,plant_type,season,short_category
2004-06-02T15:11:26,HERB 3.5,18,,2004-06-02 15:11:26,,postgres,herb,none,HR3
```
When exporting the line_id takes on the value of ts_update. This happens across multiple export formats.

**Comment by kennethreitz** [comment]
It's very hard to parse the data that you provided. Can you format it correctly?

**Comment by aklaver** [comment]
Well the csv is as produced by records, so I assumed it was formatted correctly. I am not sure what I can do to make it better.

**Comment by kennethreitz** [comment]
@aklaver I meant the formatting of the issue. i cleaned it up for you :)

**Comment by aklaver** [comment]
Aah, forgot Markdown is supported, will remember next time. Thanks.

**Comment by aklaver** [comment]
Just checking on the status of this.  I would be willing to some legwork if you could give me a push in the right direction.

**Comment by kennethreitz** [comment]
@aklaver that would be great!

It's not very clear to me what the actual issue is that you're experiencing. But, if you can reproduce it, it should be relatively simple to fix (the codebase is quite small).

**Comment by aklaver** [comment]
Well that was easy enough. I went to create a test case and figured I ought to start with the latest code:

records==0.4.3
tablib==0.11.2

I ran the same query as above and now returns correctly:
`
print rs.export('csv')
line_id,category,cell_per,ts_insert,ts_update,user_insert,user_update,plant_type,season,short_category
4,HERB 3.5,18,,2004-06-02T15:11:26,,postgres,herb,none,HR3
5,H PREM 3.5,18,,2004-06-02T15:11:26,,postgres,herb,none,HP3
7,HERB G,1,,2004-06-02T15:11:26,,postgres,herb,none,HG
9,HERB TOP,1,,2004-06-02T15:11:26,,postgres,herb,none,HT

`
In others word I get an integer in the first column instead of a datetime. Seems this is fixed.
